index.html

```diff
@@ -9,6 +9,17 @@
   <link rel="stylesheet" href="css/modules.css">
   <link rel="stylesheet" href="css/style.css">
   <link rel="stylesheet" href="css/normalize.css">
+  <script id="projects-template" type="text/x-handlebars-template">
```

Comment:
Really clean. Nice job with this section. Everything is well named and has a purpose. No corpse code. Great job!